global/spotlight.scr

```diff
@@ -36,24 +36,28 @@
 
 main:
 
-	level waittill prespawn
+	if(level.gametype == 0)
```

Comment:
Level waittill prespawn should still be done in multiplayer
regardless if it throws a error